src/library/Uno.Themes.WinUI.Markup/Theme/DatePicker.cs

```diff
@@ -0,0 +1,154 @@
+
+						[ResourceKeyDefinition(typeof(Brush), "DatePickerButtonBackgroundDisabled")]
+						public static ResourceValue<Brush> Disabled => new("DatePickerButtonBackgroundDisabled", true);
+					}
+
+					public static class Foreground
+					{
+						[ResourceKeyDefinition(typeof(Brush), "DatePickerButtonForeground")]
+						public static ResourceValue<Brush> Default => new("DatePickerButtonForeground", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "DatePickerButtonForegroundPointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("DatePickerButtonForegroundPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "DatePickerButtonForegroundPressed")]
+						public static ResourceValue<Brush> Pressed => new("DatePickerButtonForegroundPressed", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "DatePickerButtonForegroundDisabled")]
+						public static ResourceValue<Brush> Disabled => new("DatePickerButtonForegroundDisabled", true);
+					}
+
+					public static class BorderBrush
+					{
+						[ResourceKeyDefinition(typeof(Brush), "DatePickerButtonBorderBrush")]
+						public static ResourceValue<Brush> Default => new("DatePickerButtonBorderBrush", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "DatePickerButtonBorderBrushPointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("DatePickerButtonBorderBrushPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "DatePickerButtonBorderBrushPressed")]
+						public static ResourceValue<Brush> Pressed => new("DatePickerButtonBorderBrushPressed", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "DatePickerButtonBorderBrushDisabled")]
+						public static ResourceValue<Brush> Disabled => new("DatePickerButtonBorderBrushDisabled", true);
+					}
+
+					public static class ContainerCornerRadius
+					{
+						[ResourceKeyDefinition(typeof(CornerRadius), "DatePickerFlyoutPresenterCornerRadius")]
+						public static ResourceValue<CornerRadius> Default => new("DatePickerFlyoutPresenterCornerRadius", true);
+					}
+				}
+
+				public static class ButtonDateText
+				{
+					public static class Foreground
+					{
+						[ResourceKeyDefinition(typeof(Brush), "DatePickerButtonDateTextForeground")]
+						public static ResourceValue<Brush> Default => new("DatePickerButtonDateTextForeground", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "DatePickerButtonDateTextForegroundPointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("DatePickerButtonDateTextForegroundPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "DatePickerButtonDateTextForegroundPressed")]
+						public static ResourceValue<Brush> Pressed => new("DatePickerButtonDateTextForegroundPressed", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "DatePickerButtonDateTextForegroundDisabled")]
+						public static ResourceValue<Brush> Disabled => new("DatePickerButtonDateTextForegroundDisabled", true);
+					}
+				}
```

Comment:
This can be moved to a `DateTextForeground` class inside of the existing `Button` static class